05.ls/ls2.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+class Ls
+  COLUMN = 3
+  def initialize
+    opt = ARGV.getopts('a')
+    @directories = if opt['a']
+                     Dir.glob('*', File::FNM_DOTMATCH)
+                   else
+                     Dir.glob('*')
+                   end
```

Comment:
このままでもいいですが、`if`と`else`の中身が簡単なので三項演算子で書いちゃってもいいかもしれませんね。参考までに。

```suggestion
    @directories = if opt['a'] ? Dir.glob('*', File::FNM_DOTMATCH) : Dir.glob('*')
```

Reply:
このような書き方もあるんですね！
今後は条件分岐が2つだけの時は、3項演算子も活用していこうと思います。